distribution/src/main/packaging/scripts/postrm

```diff
@@ -76,6 +77,12 @@ if [ "$REMOVE_DIRS" = "true" ]; then
         echo " OK"
     fi
 
+    if [ -d "$MODULES_DIR" ]; then
```

Comment:
I agree - I just don't know how to do with nebula.ospackage. Also, it seems that the current configuration leaves empty directory around like `/usr/share/elasticsearch/bin`, `/usr/share/elasticsearch/lib` or `/usr/share/elasticsearch/modules`. 

In the case of modules and if empty directories remains on the filesystem, the plugin service will fail to find plugin descriptor in empty directories like `/usr/share/elasticsearch/modules/deprecated-plugin`... 

That's why I chose this option.